### ISSUE TITLE
Session config from web.xml not passed to undertow

### ISSUE THREAD

**Issue by sc3n** [quarkusio/quarkus#13584]
**Describe the bug**
Session config is not read from `web.xml`.

**Expected behavior**
Session config is read from `web.xml`.
`HttpOnly` attribute of cookie created by `GreetingResource` is `true`.

**Actual behavior**
`HttpOnly` attribute of cookie created by `GreetingResource` is `false`.

**To Reproduce**

See attached archive: [code-with-quarkus.zip](https://github.com/quarkusio/quarkus/files/5622686/code-with-quarkus.zip)

1. Start application
2. `GET hello-resteasy`
3. Analyse cookie

**Configuration**
web.xml:
```xml
<web-app>
	<session-config>
		<session-timeout>10</session-timeout>
		<cookie-config>
			<http-only>true</http-only>
			<secure>false</secure>
		</cookie-config>
		<tracking-mode>COOKIE</tracking-mode>
	</session-config>
</web-app>
```

**Environment**
- Windows
- Java 11
- mvn
- Quarkus 1.10.2.Final

**Additional context**
See related question on stackoverflow, which was not created by myself: https://stackoverflow.com/questions/61165144/quarkus-changing-the-default-session-cookie-name

**Comment by sberyozkin** [comment]
CC @stuartwdouglas

**Comment by stuartwdouglas** [comment]
As a workaround you can use a `ServletContainerInitializer` to set the session config programatically.

**Comment by stuartwdouglas** [comment]
Also this was fixed in 1.11 in f89e05920d2586c42ce8f556e3752a8d9b4cb923 , if you still have problems this is a different issue.

**Comment by gsmet** [comment]
@autumnfound if something is not working for you, please open a new issue with a reproducer. Thanks!
